how-to-guides/celestia-node-troubleshooting.md

```
@@ -311,3 +311,68 @@ sudo systemctl daemon-reload
+
+When a bridge or full storage node runs on `ext4` file system, there will be errors like the following due to large folder size:
+
+```bash
+ERROR   header/sync     sync/sync.go:227        syncing headers {"from": 5074102, "to": 5161144, "err": "creating file: creating ODSQ4 file: creating Q4 file: creating Q4 file: open /root/.celestia-bridge-mocha-4/blocks/C9ADF6D9F862D92993D67977DE407D17ECF7F1DACE5FB7FE9A6845F4BD0172CE.q4: no space left on device"}
+```
+
+You may also observe similar errors from the `tar` command when extracting a snapshot of an archival bridge node.
+
+There are two solutions to this problem:
+
+#### Option 1: enable the large_dir feature for ext4
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)